db/migrate/20240207101439_create_admins.rb

```diff
@@ -0,0 +1,15 @@
+class CreateAdmins < ActiveRecord::Migration[7.1]
+  def change
+    create_table :admins do |t|
+      t.string "email", null: false, limit: 64
+      t.string "full_name", null: false, limit: 64
+      t.string "feature_flag_id"
```

Comment:
Since this controls the flow through the journey primarily I don’t think this should be a flag for admins and should remain exclusively the domain of Users and NullUser

Reply:
Fixed and force pushed, thank you 🙂